### ISSUE TITLE
Update for purescript v0.7.0.0

### ISSUE THREAD

**Issue by shinnn** [purescript-contrib/node-purescript-bin#4]
https://github.com/purescript/purescript/releases/tag/v0.7.0

I'll take it on.

**Comment by garyb** [comment]
Great, thanks!

**Comment by paf31** [comment]
What's involved here?

- [ ] Bump version
- [ ] Change executable names and wrap `psc-bundle` and `psc-publish`

Anything else?

**Comment by garyb** [comment]
I think thats it, just need to bump the `purescript-version` field in `package.json` at the same time as the package version.

**Comment by e-jigsaw** [comment]
Hi,

What is this issue status? I tried to upgrade and send Pull Request: https://github.com/e-jigsaw/node-purescript-bin/commit/fcebed46a68f4307ed4dc632bc247488da8ac802

But, some problems happen.

1. Github release tag is `v0.7.0` but `psc --version` returns `0.7.0.0`. So test is failure. (Monkey patched: https://github.com/e-jigsaw/node-purescript-bin/commit/fcebed46a68f4307ed4dc632bc247488da8ac802#diff-1dd241c4cd3fd1dd89c570cee98b79ddR100)
2. `psc-publish --help` returns

```
There is a problem with your package, which meant that it could not be
published.
Details:
  The bower.json file was not found. Please create one, or run `pulp init`.
```

This is PureScript's bug...? `psc-publish --version` also returns above message.

Thanks

**Comment by paf31** [comment]
I don't think `psc-publish` supports `--help`.

**Comment by shinnn** [comment]
Can I ask why `psc-publish` doesn't support `--help` and `--version`?

**Comment by paf31** [comment]
I think it's just an oversight.

**Comment by e-jigsaw** [comment]
:+1:

**Comment by shinnn** [comment]
Released v0.7.0. Thanks for your patience.
